src/LibraryManager.Vsix/Resources/Text.resx

```diff
@@ -163,10 +163,10 @@ Do you want to continue?</value>
     <value>&lt;Choose a library to select files to install&gt;</value>
   </data>
   <data name="DisableRestoreOnBuild" xml:space="preserve">
-    <value>Disable Restore on Build</value>
+    <value>Disabl&amp;e Restore on Build</value>
   </data>
   <data name="EnableRestoreOnBuild" xml:space="preserve">
-    <value>Enable Restore Client-Side Libraries on Build</value>
+    <value>&amp;Enable Restore Client-Side Libraries on Build</value>
```

Comment:
>Enable Restore Client-Side Libraries on Build [](start = 16, length = 45)

Shouldn't this message be worded in the same fashion as DisableRestoreOnBuild's?

Reply:
Yes. Good point. Thanks Todd.

Reply:
Changed wording of "Disable" to match the "Enable" wording.
Note: Disable has no dialog, so has no ellipses.